apiary_v2.apib

```diff
@@ -449,11 +449,14 @@ This lets you:
                 "id": "8a2b6517-e020-46fd-8cd0-ec441d4ac4b4",
                 "source_design_id": "5e39f63b-a084-4d80-ae29-1fdb87a79922",
                 "product": "<Unmade Editor URL>",
-                "preview": "<URL to the main rendering of the design>",
                 "description": "<Description of the product and design>",
-                "secondary_previews": [
+                "previews": [
+                    {
+                        "name": "front",
+                        "image": "https://PREVIEW_URL"
+                    },
                     {
-                        "name": "right",
```

Comment:
nice! I think below (not important right now) there is a mix & match about colours used for placements and design elements. As I said not important right now